localized/games/eggquiz/eggquizmath_levels_hi.tsv

```diff
@@ -427,27 +427,27 @@ SpecialTest_25	1	250001, 250002, 250003, 250004, 250005
-			270002	word_problem	एली ने आम के २ बोक्स खरीदे। प्रत्येक बोक्स में ४ आम हैं। उसने कितने आम खरीदे?	एली ने आम के २ बोक्स खरीदे। प्रत्येक बोक्स में ४ आम हैं। उसने कितने आम खरीदे?				8				
+			270001	word_problem	दर्पण के पास ३ जोड़ी मोजे हैं। प्रत्येक जोड़ी में २ मोजे हैं। दर्पण के पास कितने मोजे हैं?	दर्पण के पास ३ जोड़ी मोजे हैं। प्रत्येक जोड़ी में २ मोजे हैं। दर्पण के पास कितने मोजे हैं?				6				
+			270002	word_problem	सपना ने आम के २ बोक्स खरीदे। प्रत्येक बोक्स में ४ आम हैं। उसने कितने आम खरीदे?	सपना ने आम के २ बोक्स खरीदे। प्रत्येक बोक्स में ४ आम हैं। उसने कितने आम खरीदे?				8				
 			270003	word_problem	चरागाह में ३ भेड़ें हैं। प्रत्येक भेड़ के ४ बच्चे होते हैं। चरागाह में कितने मेमने हैं?	चरागाह में ३ भेड़ें हैं। प्रत्येक भेड़ के ४ बच्चे होते हैं। चरागाह में कितने मेमने हैं?				12				
-			270004	word_problem	जेन ने फ्लावर पॉट में ३ फूलों की गिनती की। प्रत्येक फूल में ५ पंखुड़यिाँ होती हैं। सभी में कितनी पंखुड़यिाँ हैं?	जेन ने फ्लावर पॉट में ३ फूलों की गिनती की। प्रत्येक फूल में ५ पंखुड़यिाँ होती हैं। सभी में कितनी पंखुड़यिाँ हैं?				15				
+			270004	word_problem	अलका ने फ्लावर पॉट में ३ फूलों की गिनती की। प्रत्येक फूल में ५ पंखुड़यिाँ होती हैं। सभी में कितनी पंखुड़यिाँ हैं?	अलका ने फ्लावर पॉट में ३ फूलों की गिनती की। प्रत्येक फूल में ५ पंखुड़यिाँ होती हैं। सभी में कितनी पंखुड़यिाँ हैं?				15				
 			270005	word_problem	खेत पर २ वैगन हैं। प्रत्येक वैगन में ८ बच्चे होते हैं। कितने बच्चे वैगनों में हैं?	खेत पर २ वैगन हैं। प्रत्येक वैगन में ८ बच्चे होते हैं। कितने बच्चे वैगनों में हैं?				16				
 			270006	word_problem	एक मधुमक्खी के ६ पैर होते हैं। २ मधुमक्खियों के कितने पैर होते हैं?	एक मधुमक्खी के ६ पैर होते हैं। २ मधुमक्खियों के कितने पैर होते हैं?				12				
 			270007	word_problem	एक बतख के २ पैर होते हैं। ३ बतख के कितने पैर हैं?	एक बतख के २ पैर होते हैं। ३ बतख के कितने पैर हैं?				6				
-			270008	word_problem	शेल्ली ने डोनट्स के ४ बोक्स खरीदे। प्रत्येक बोक्स में ५ डोनट्स होते हैं। उसने कुल कितने डोनट्स खरीदे?	शेल्ली ने डोनट्स के ४ बोक्स खरीदे। प्रत्येक बोक्स में ५ डोनट्स होते हैं। उसने कुल कितने डोनट्स खरीदे?				20				
+			270008	word_problem	नेहा ने डोनट्स के ४ बोक्स खरीदे। प्रत्येक बोक्स में ५ डोनट्स होते हैं। उसने कुल कितने डोनट्स खरीदे?	नेहा ने डोनट्स के ४ बोक्स खरीदे। प्रत्येक बोक्स में ५ डोनट्स होते हैं। उसने कुल कितने डोनट्स खरीदे?				20				
 			270009	word_problem	सैली को एक मील चलने के लिए ७ मिनट लगते हैं। ४ मील चलने में उसे कितना समय लगेगा?	सैली को एक मील चलने के लिए ७ मिनट लगते हैं। ४ मील चलने में उसे कितना समय लगेगा?				28				
-			270010	word_problem	किट रूथ और डीन ने पिज़्ज़ा के २ स्लाइस खाए। पिज़्ज़ा के कितने स्लाइस उन्होंने सभी में खाए?	किट रूथ और डीन ने पिज़्ज़ा के २ स्लाइस खाए। पिज़्ज़ा के कितने स्लाइस उन्होंने सभी में खाए?				6				
+			270010	word_problem	शालिनी रूथ और रमेश ने पिज़्ज़ा के २ स्लाइस खाए। पिज़्ज़ा के कितने स्लाइस उन्होंने सभी में खाए?	शालिनी रूथ और रमेश ने पिज़्ज़ा के २ स्लाइस खाए। पिज़्ज़ा के कितने स्लाइस उन्होंने सभी में खाए?				6				
```

Comment:
Just check `किट रूथ` is the person name. If so update `शालिनी रूथ`

Reply:
किट रूथ और डीन are three different persons.